starlite/testing/sync_test_client/client.py

```diff
@@ -117,28 +114,6 @@ def session(self) -> "CookieBackend":
             )
         return self.session_backend
 
-    # @property
-    # def session_backend(self) -> "BaseSessionBackend":
-    #     if not self._session_backend:
-    #         raise ImproperlyConfiguredException(
-    #             "Session has not been initialized for this TestClient instance. You can"
-    #             "do so by passing a configuration object to TestClient: TestClient(app=app, session_config=...)"
-    #         )
-    #     return self._session_backend
-
-    # @contextmanager
-    # def portal(self) -> Generator["BlockingPortal", None, None]:
```

Comment:
why is this commented out?

Reply:
@Goldziher It is now in the `BaseTestClient` class

Reply:
ok, then pls clean it